--- nagios.cfg.orig	2008-09-23 12:18:03.000000000 +0200
+++ nagios.cfg	2008-09-25 15:19:07.000000000 +0200
@@ -22,7 +22,10 @@
 # Debian uses by default a configuration directory where nagios3-common,
 # other packages and the local admin can dump or link configuration
 # files into.
-cfg_dir=/etc/nagios3/conf.d
+#cfg_dir=/etc/nagios3/conf.d
+
+# ConfigMgr-generated conf file
+cfg_file=/etc/vigilo-vigiconf/prod/nagios.cfg
 
 # OBJECT CONFIGURATION FILE(S)
 # These are the object configuration files in which you define hosts,
@@ -141,7 +144,7 @@
 # you will have to enable this.
 # Values: 0 = disable commands, 1 = enable commands
 
-check_external_commands=0
+check_external_commands=1
 
 
 
@@ -247,6 +250,7 @@
 
 #broker_module=/somewhere/module1.o
 #broker_module=/somewhere/module2.o arg1 arg2=3 debug=0
+broker_module=/usr/lib/ndoutils/ndomod-mysql-3x.o config_file=/etc/nagios3/ndomod.cfg
 
 
 
@@ -275,7 +279,7 @@
 # If you want messages logged to the syslog facility, as well as the
 # Nagios log file set this option to 1.  If not, set it to 0.
 
-use_syslog=1
+use_syslog=0
 
 
 
@@ -320,7 +324,7 @@
 # statistics reporting, you do not need to enable this option.  In
 # this case, set the value to 0.
 
-log_initial_states=0
+log_initial_states=1
 
 
 
@@ -340,7 +344,7 @@
 # this value to 0.  If passive checks should be logged, set
 # this value to 1.
 
-log_passive_checks=1
+log_passive_checks=0
 
 
 
@@ -381,7 +385,7 @@
 # program start time that an initial check of all services should
 # be completed.  Default is 30 minutes.
 
-max_service_check_spread=30
+max_service_check_spread=5
 
 
 
@@ -422,7 +426,7 @@
 # program start time that an initial check of all hosts should
 # be completed.  Default is 30 minutes.
 
-max_host_check_spread=30
+max_host_check_spread=5
 
 
 
@@ -442,7 +446,7 @@
 # This is the frequency (in seconds!) that Nagios will process
 # the results of host and service checks.
 
-check_result_reaper_frequency=10
+check_result_reaper_frequency=3
 
 
 
@@ -601,8 +605,8 @@
 # ocsp command, and performance data commands.  All values are in
 # seconds.
 
-service_check_timeout=60
-host_check_timeout=30
+service_check_timeout=30
+host_check_timeout=10
 event_handler_timeout=30
 notification_timeout=30
 ocsp_timeout=5
@@ -721,7 +725,7 @@
 # on what aggressive host check is or check out the source code in
 # base/checks.c
 
-use_aggressive_host_checking=0
+use_aggressive_host_checking=1
 
 
 
@@ -798,7 +802,7 @@
 # performance data.
 # Values: 1 = process performance data, 0 = do not process performance data
 
-process_performance_data=0
+process_performance_data=1
 
 
 
@@ -982,7 +986,7 @@
 # check the "freshness" of service check results.  If you have
 # disabled service freshness checking, this option has no effect.
 
-service_freshness_check_interval=60
+service_freshness_check_interval=10
 
 
 
@@ -993,7 +997,7 @@
 # manner.
 # Values: 1 = enabled freshness checking, 0 = disable freshness checking
 
-check_host_freshness=0
+check_host_freshness=1
 
 
 
@@ -1002,7 +1006,7 @@
 # check the "freshness" of host check results.  If you have
 # disabled host freshness checking, this option has no effect.
 
-host_freshness_check_interval=60
+host_freshness_check_interval=10
 
 
 
@@ -1029,7 +1033,7 @@
 # Values: 1 = enable flap detection
 #         0 = disable flap detection (default)
 
-enable_flap_detection=1
+enable_flap_detection=0
 
 
 
@@ -1054,7 +1058,7 @@
 #	strict-iso8601	(YYYY-MM-DDTHH:MM:SS)
 #
 
-date_format=iso8601
+date_format=euro
 
 
 
